### PR TITLE
feature/MRTI-3390 # add bulk lookup in CLI

### PR DESCRIPTION
**Pull request by juan-kabbali**
The command was added as `bulk_lookup_threats `

```
ocd-dtl bulk_lookup_threats < VAL ... > [--ip VAL ... | --file VAL ... | --domain VAL ...] -i [atomtype:path/to/file.txt ...]
```

For more information about how to use it go [here](https://github.com/cert-orangecyberdefense/datalake/blob/9ee7d59275098883ef2532cd378da0a57b4c4401/docs/Bulk_lookup_threats.md)

### CODE REVIEW

**Review by joginsky — APPROVED** [review]

File: docs/Bulk_lookup_threats.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+  ```shell
+  $ ocd-dt bulk_lookup_threats 113.223.40.103 5.78.23.158 reverso.net f0f5d30e91006c8ca7a528f26432ftt5 
+  ```     
+  
+
+With all that in mind, you can combine them to fit your needs:
+```shell
+$ ocd-dt bulk_lookup_threats reverso.net 113.223.40.103 --domain paiza.com --ip 45.96.65.132 -i ip:path/to/file/myiplist.txt -i path/to/file/myfilelist.txt
+                             ----untyped domain ip----- ---typed domain--- ----typed ip----  -------typed file as ip-------- ---------untyped file---------                                                
+```
```
Comment (Hugo-C) [inline docs/Bulk_lookup_threats.md:52]: :+1:

File: docs/Bulk_lookup_threats.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+    > -ad, --atom-details  
+    returns threats full details
+   
+    > -i INPUT, --input INPUT  
+    read threats to add from FILE.
+  
+* Typed atom flags
+    > --apk APK             
+    --asn ASN             
+    -cc CC
```
Comment (Hugo-C) [inline docs/Bulk_lookup_threats.md:81]: typo: missing "-"

File: datalake_scripts/scripts/bulk_lookup_threats.py [hunk 3]
```diff
@@ -0,0 +1,202 @@
...
+        '-i',
+        '--input',
+        action='append',
+        help='read threats to add from FILE. [atomtype:path/to/file.txt]',
+    )
+    parser.add_argument(
+        '-ot',
+        '--output-type',
+        help='set to the output type desired {json,csv}. Default is human readable',
+    )
```
Comment (Hugo-C) [inline datalake_scripts/scripts/bulk_lookup_threats.py:59]: Default is json.
Also please add it to Bulk_lookup_threats.md

**Review by Hugo-C — CHANGES_REQUESTED** [review]

File: datalake_scripts/scripts/bulk_lookup_threats.py [hunk 4]
```diff
@@ -0,0 +1,193 @@
...
+    boolean_to_text_and_color = {
+        True: ('FOUND', '\x1b[6;30;42m'),
+        False: ('NOT_FOUND', '\x1b[6;30;41m')
+    }
+
+    for atom_type in raw_response.keys():
+        logger.info(f'{blue_bg}{"#" * 60} {atom_type.upper()} {"#" * (60 - len(atom_type))}{eol}')
+
+        for atom in raw_response[atom_type]:
+            found = atom.get('threat_found', True) if 'threat_found' in atom.keys() else True
```
Comment (Hugo-C) [inline datalake_scripts/scripts/bulk_lookup_threats.py:185]: `atom.get('threat_found', True)` should be enough no ? no need for the if/else

**Review by Hugo-C — CHANGES_REQUESTED** [review]
Please look at:
* https://github.com/cert-orangecyberdefense/datalake/pull/18#discussion_r633621488
* https://github.com/cert-orangecyberdefense/datalake/pull/18#discussion_r633557072
* (Optionally) https://github.com/cert-orangecyberdefense/datalake/pull/18#discussion_r633606173

**Review by Hugo-C — APPROVED** [review]

**Comment by Hugo-C** [comment]
Remember to bump setup.py before merging :)

**Pull request merged**